cluster/saltbase/salt/etcd/etcd.manifest

```diff
@@ -1,3 +1,19 @@
+{% set cluster_state = 'new' -%}
+{% set host_name = pillar.get('master_name', '') -%}
+{% set etcd_cluster_array = pillar.get('initial_etcd_cluster', '').split(',') -%}
+{% set etcd_cluster = '' -%}
+{% set vars = {'etcd_cluster': '', 'cluster_state': cluster_state} -%}
```

Comment:
Because jinja defines new variables inside for loop that hide variables from outside the loop.